deps/build.jl

```diff
@@ -1,31 +1,57 @@
-using BinDeps, Compat
-@BinDeps.setup
+using Compat
 
-@osx_only Libdl.dlopen("libstdc++",Libdl.RTLD_GLOBAL)
+import Compat: String
 
-cpxvers = ["124","125","1251","1260","1261","1262","1263"]
+depsfile = joinpath(dirname(@__FILE__),"deps.jl")
+if isfile(depsfile)
+    rm(depsfile)
+end
+
+function write_depsfile(path)
+    f = open(depsfile,"w")
+    println(f,"const libcplex = \"$path\"")
+    close(f)
+end
 
-libnames = ASCIIString["cplex"]
-for v in cpxvers
-    push!(libnames, "cplex$v")
-    push!(libnames, "libcplex$v.so")
-    push!(libnames, "libcplex$v.dylib")
+if is_apple()
+	Libdl.dlopen("libstdc++",Libdl.RTLD_GLOBAL)
```

Comment:
I wonder whether this will be around forever. Will CPLEX on mac eventually move to using clang/libc++ ? Maybe they already have?